src/client/client4.js

```diff
@@ -2276,8 +2276,14 @@ export default class Client4 {
         );
     };
 
-    // Role Routes
+    getUserAnalytics = async (uid, teamId, types) => {
+        return this.doFetch(
+            `${this.getBaseRoute()}/useranalytics${buildQueryString({uid, teamId, types})}`,
```

Comment:
I really don't think that this should be using the base route. I think perhaps `getTeamMemberRoute` would be much better perhaps even
```
`${this.getTeamMemberRoute(teamId, userId)}/memberconnections${buildQueryString({types})}`
```
Although since teams don't have many queries like this perhaps just `getUserRoute(userId)`

I still haven't figured out what the argument `types` is for and it isn't described anywhere I've looked so far.

Reply:
Route has been updated.